### PR TITLE
error fild to city, street, building and apartment was added

### PR DESCRIPTION
**Pull request by Skydry333**
## Description
Error message is not displayed when typing inconsistent values in the fields city, street, house and apartment of Checkout Process page. I also added tests for checkout-form

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** до ** | ** після ** |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Comment by Skydry333** [comment]
in the input field "building" we should not limit the user only to numbers, because the name of the house is also composed of letters

**Review by maarekua — DISMISSED** [review]

File: src/containers/checkout/checkout-form/delivery-type/delivery-type.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const DeliveryType = ({ setFieldValue, touched, errors, deliveryType, setDeliver
         setDeliveryType(deliveryTypeValue);
       }
       setDeliveryTypeToStorage(deliveryTypeValue);
+      // console.log(setDeliveryTypeToStorage(deliveryTypeValue));
```
Comment (maarekua) [inline src/containers/checkout/checkout-form/delivery-type/delivery-type.js:30]: Remove console.log

File: src/containers/checkout/checkout-form/tests/checkout-form.spec.js [hunk 2]
```diff
@@ -33,9 +61,19 @@ const userData = {
...
+  errors: {},
+  handleBlur: mockBlur,
+  resetForm: mockReset,
+  handleSubmit: mockSubmit,
+  handleChange: mockChange
+};
 
 useDispatch.mockImplementation(() => dispatch);
 useSelector.mockImplementation(() => userData);
+useFormik.mockImplementation(() => formik);
```
Comment (maarekua) [inline src/containers/checkout/checkout-form/tests/checkout-form.spec.js:76]: We don't need to mock 'formik'. Here are instructions on how to test formik.

https://testing-library.com/docs/example-react-formik/

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/checkout/checkout-form/delivery-type/delivery-type.js (hunk 1 above)
Comment (Skydry333) [reply src/containers/checkout/checkout-form/delivery-type/delivery-type.js:30]: ок

File: src/containers/checkout/checkout-form/tests/checkout-form.spec.js (hunk 2 above)
Comment (Skydry333) [reply src/containers/checkout/checkout-form/tests/checkout-form.spec.js:76]: I'm already working on it

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by tarasSanotskyi — APPROVED** [review]

**Pull request merged**